sass/layouts/_card-section.scss

```diff
@@ -1,8 +1,379 @@
 section.blank {
-    background-color: $color-secondary-400 !important;
-    width: 100%;
-    height: 30%;
+    margin: 0;
+    padding: 0;
     .blank--content {
+        background: $color-secondary-400 !important;
+        width: 100%;
+        height: 40vh;
         color: #FFF;
     }
 }
+
+section.card {
```

Comment:
This is repetitive. Better way is to give the body of this `html` a class and nest other styles related to that pages within that class.

i.e

```
.card-section {
    // Then you can nest properly. This will `give .card-section .blank`
    .blank {
    }
    .card {
    }
}
```

Sounds good?

Reply:
Okay, I will correct all tomorrow